### PR TITLE
update sources

### PR DESCRIPTION
**Pull request by snafuy**
sources added since previous update

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,7 @@
 local server.bat
 todo.txt
 build.js
-
+build*
```
Comment (Asmor) [inline .gitignore]: This is no good. I use github for deployment of KFC (yes, I know it's not a good practice, but it's what I'm doing), so build must be included.

**Comment by Asmor** [comment]
Looks ok other than that change to `.gitignore`. Please revert that and I can accept this.

**Comment by snafuy** [comment]
Whoops. Still a lot I need to learn about Git. I assumed that changes I made AFTER the pull request wouldn't get added to it retroactively.

**Comment by snafuy** [comment]
Ok, good to go, more commits waiting on a branch until this one is done.

**Comment by Asmor** [comment]
I went ahead and merged this so you're not blocked, but I won't be able to actually deploy until late tonight at the earliest.

**Pull request merged**